ghcide/session-loader/Development/IDE/Session.hs

```diff
@@ -619,11 +621,15 @@ cradleToOptsAndLibDir logger cradle file = do
                 CradleFail err       -> return (Left [err])
                 -- For the None cradle perhaps we still want to report an Info
```

Comment:
Now you're doing it, so you could get rid of these comments?

Reply:
Sure :+1: Fixed and squashed.